extension/js/common/api/account-servers/flowcrypt-com-api.ts

```diff
@@ -20,7 +20,7 @@ export type SubscriptionInfo = { level?: SubscriptionLevel; expired?: boolean };
 
 export namespace BackendRes {
   export type FcAccountLogin = { registered: boolean, verified: boolean };
-  export type FcAccount$info = { alias: string, email: string, intro: string, name: string, photo: string, default_message_expire: number };
+  export type FcAccount$info = { alias?: string | null, default_message_expire: number };
```

Comment:
These were unused, after we moved Encrypted Contact Page setup to flowcrypt.com/account/